### ISSUE TITLE
Configuration example in readme

### ISSUE THREAD

**Issue by MichaelJCole** [pouchdb/express-pouchdb#119]
Hi, cool project.  I'm excited to use it.

Tried this in config.json, but it didn't do anything.  What did I miss?

```
{
  "couchdb": {
    "database_dir": "pouchdb_data"
  }
}
```
and
```
{
    "database_dir": "pouchdb_data"
}
```

Thanks!

**Comment by marten-de-vries** [comment]
#162 explains that the available configuration options can be determined using Fauxton. README.md already included a ``.defaults()`` example that changed the database directory. pouchdb-server now supports couchdb.database_dir as a setting, which is documented via its command line interface (and thus also in its README.md)

**Comment by marten-de-vries** [comment]
Fixed in 29c016b28d93b7dad52d0df5a35bea6ec493da29
